lighthouse-core/test/results/sample_v2.json

```diff
@@ -2114,16 +2150,16 @@
                 "width": 16,
                 "height": 16
               },
-              "snippet": "<img src=\"blob:http://localhost:10200/822c70a0-b912-41c7-9a21-56c3d309e75b\">",
+              "snippet": "<img src=\"blob:http://localhost:10200/5a659d35-80ca-42e2-8f98-19daed3b98aa\">",
```

Comment:
you got some churn here to manually remove

Reply:
> you got some churn here to manually remove

it's only the four urls, so maybe just an optional nit?